components/dashboard/src/settings/Plans.tsx

```diff
@@ -294,8 +339,8 @@ export default function () {
     } else {
         const targetPlan = freePlan;
         let bottomLabel;
-        if (scheduledDowngradePlanId === targetPlan.chargebeeId) {
-            bottomLabel = <p className="text-green-600">Downgrade scheduled<br/><a className="text-blue-light leading-6" href="javascript:void(0)">Cancel</a></p>;
+        if (scheduledDowngradePlanId === targetPlan.chargebeeId && !pendingDowngradeCancellation) {
+            bottomLabel = <p className="text-green-600">Downgrade scheduled<br/><a className="text-blue-light leading-6" href="javascript:void(0)" onClick={() => confirmCancelDowngrade()}>Cancel</a></p>;
```

Comment:
**suggestion**: In addition to this, disabling all the buttons on all other cards could make sense as they are all broken during an upgrade or a downgrade process. However, this could be considered out of the scope of this PR and a good follow-up issue. 🏓